scripts/sct_maths.py

```diff
@@ -515,16 +523,34 @@ def compute_similarity(data1, data2, fname_out='', metric='', verbose=1):
     :param metric: 'mi' for mutual information or 'corr' for pearson correlation coefficient
     :return: None
     '''
+    import matplotlib
```

Comment:
ok i'll change it-- can you please put a note to discuss that during next sct meeting-- thanks